src/parse_instruction.py

```diff
@@ -350,6 +371,24 @@ def expect(n: str) -> str:
             else:
                 op = expect("&+-*")
 
+            # Parse `sym-_SDA_BASE_(r13)` as a Macro, equivalently to `sym@sda21(r13)`
+            if tok == "-" and arg_elems[0] == "_":
+                reloc_name = parse_word(arg_elems)
+                if reloc_name not in ("_SDA_BASE_", "_SDA2_BASE_"):
+                    raise DecompFailure(
+                        f"Unexpected symbol {reloc_name} in subtraction expression"
+                    )
+                assert value
+                macro = Macro("sda21", value)
+                if not arg_elems:
+                    return macro
+                expect("(")
+                rhs = parse_arg_elems(arg_elems, arch)
+                assert isinstance(rhs, Register)
+                assert rhs in [Register("r2"), Register("r13")]
+                expect(")")
+                return AsmAddressMode(macro, rhs)
```

Comment:
```suggestion
                value = Macro("sda21", value)
```
the existing code should be able to turn the value into an AsmAddressMode if needed I think

(+ needs an else clause or moving the `if tok == "-" and arg_elems[0] == "_":` check to the outer elif chain)